components/wrist.py

```diff
@@ -35,7 +35,7 @@ def __init__(self):
         self.wrist_controller = self.wrist.getClosedLoopController()
 
         wrist_config = SparkMaxConfig()
-        wrist_config.inverted(True)
+        wrist_config.inverted(False)
         wrist_config.setIdleMode(SparkMaxConfig.IdleMode.kBrake)
         wrist_config.closedLoop.P(
             0.64571,
```

Comment:
Probably worth plugging these new feedforwards into SysId to get new PID gain estimates.